tcpdf.php

```diff
@@ -5469,27 +5469,6 @@ protected function getCellCode($w, $h=0, $txt='', $border=0, $ln=0, $align='', $
 			$xdk = $xdx * $k;
 			// print text
 			$s .= sprintf('BT %F %F Td [(%s)] TJ ET', $xdk, (($this->h - $basefonty) * $k), $txt2);
-			if (isset($uniblock)) {
```

Comment:
Why removing this code?

Reply:
`$uniblock` is never set - please advise

Reply:
The code might be usefull for future/not yet implemented overlapping characters printing thus I added `@phpstan-ignore-line` only